src/components/_datepicker-demo/index.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from "react";
+
+import { registerLocale } from "react-datepicker";
+
+import DateTimePicker from "../DateTimePicker";
+import Custom from "./CustomInput";
+
+import enUs from "date-fns/locale/en-US";
+import ru from "date-fns/locale/ru";
+import fr from "date-fns/locale/fr";
+import hy from "../DateTimePicker/_helpers/custom_locale/hy";
+
+registerLocale("en", enUs);
+registerLocale("ru", ru);
+registerLocale("fr", fr);
+registerLocale("hy", hy);
+
+const Demo = props => {
+  const [date, setDate] = useState(null);
+  const [lang, setLang] = useState("en");
+
+  return (
+    <div style={{ padding: "20px" }}>
+      <div style={{ margin: "20px" }}>
+        <span>Language:</span>
+        <select style={{ fontSize: '16px' }} onInput={e => setLang(e.target.value)}>
+          <option value="en">English</option>
+          <option value="ru">Russian</option>
+          <option value="fr">French</option>
+          <option value="hy">Armenian</option>
+        </select>
+      </div>
+      <div style={{ margin: "20px" }}>
+        <DateTimePicker
```

Comment:
Our goal is to have datepicker and time picker represented as on component. It can take property like which will allow us to render it as date only, time only or date time. We can decide how to proceed with styling later.

Reply:
Added type prop, to specify "date", "time" or "datetime"